src/azure/Elastic.Apm.Azure.Functions/Elastic.Apm.Azure.Functions.csproj

```diff
@@ -19,8 +19,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.1.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Http.Extensions" Version="2.1.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.1.22" />
+    <PackageReference Include="Microsoft.AspNetCore.Http.Extensions" Version="2.2.0" />
```

Comment:
Should be 2.1.21 as 2.2.0 is depreciated.

Reply:
Thanks for the heads up @thompson-tomo, addressed 👍